### ISSUE TITLE
Missing string

### ISSUE THREAD

**Issue by aspark21** [kuracloud/moodle-block_kuracloud#12]
Invalid get_string() identifier: 'kuracloud:configureapi' or component 'block_kuracloud'. Perhaps you are missing $string['kuracloud:configureapi'] = ''; in /var/www/html/blocks/kuracloud/lang/en/block_kuracloud.php?

**Comment by shadowspawn** [comment]
Thanks @aspark21 

What context do you see the error in? (i.e. How do I reproduce it?)

**Comment by aspark21** [comment]
I found this while running the behat tests

The full message being

```
And I set the following system permissions of "Teacher" role:: debugging() message/s found:
Invalid get_string() identifier: 'kuracloud:configureapi' or component 'block_kuracloud'. Perhaps you are missing $string['kuracloud:configureapi'] = ''; in /var/www/html/blocks/kuracloud/lang/en/block_kuracloud.php?
line 353 of /lib/classes/string_manager_standard.php: call to debugging()
line 7401 of /lib/moodlelib.php: call to core_string_manager_standard->get_string()
line 2589 of /lib/accesslib.php: call to get_string()
line 2434 of /lib/accesslib.php: call to get_capability_string()
line 118 of /admin/roles/classes/capability_table_base.php: call to get_capability_docs_link()
line 135 of /admin/roles/classes/capability_table_with_risks.php: call to core_role_capability_table_base->display()
line 708 of /admin/roles/classes/define_role_table_advanced.php: call to core_role_capability_table_with_risks->display()
line 257 of /admin/roles/define.php: call to core_role_define_role_table_advanced->display()
 (Exception)
```

If you go to https://your.moodle.site/admin/roles/define.php?action=edit&roleid=3
You should see the missing string

<img width="637" alt="Screenshot 2021-11-04 at 22 21 24" src="https://user-images.githubusercontent.com/4015496/140428590-f468fe78-2e2b-4086-a0e3-0940182b16cf.png">

**Comment by shadowspawn** [comment]
Great info thanks @aspark21 , much appreciated.

On a related note, we have recently shipped an LTI based integration for Lt (kuraCloud) which supports Moodle and Canvas and Blackboard. We have been onboarding some early adopters and just about to start making more noise about the new integration.

This block-based project won't be getting new features, but will see about fixing this bug.

**Comment by aspark21** [comment]
Ah that's great, I think we had fed that back to the team in the UK, that this would be better suited as an LTI, will reach out to them. :)

**Comment by thepurpleblob** [comment]
Just a me too. 

Invalid get_string() identifier: 'kuracloud:configureapi' or component 'block_kuracloud'. Perhaps you are missing $string['kuracloud:configureapi'] = ''; in /app/public/blocks/kuracloud/lang/en/block_kuracloud.php?

I spotted it configuring web services

**Comment by shadowspawn** [comment]
Thanks for reports. Fixed in v1.1.1, which is a final release for this project.
